### DISCUSSION TITLE
Mimir connection to IBM Cloud COS

### DISCUSSION THREAD

**Discussion by danielescattaglia** [grafana/mimir#6046] [Help and support]
Hello. I'm trying to connect mimir to IBM S3 cloud service (COS). Do you have specific documentation? I try to use AWS syntax but it doesn't work.
Thanks

**Comment by pstibrany** [comment]
Hello.

We don't have specific documentation for using IBM S3 cloud service (COS).

If you want to share your AWS S3 syntax, we can take a look and maybe find why it doesn't work.
